src/man/sss_ssh_knownhostsproxy.1.xml

```diff
@@ -27,6 +27,14 @@
         </cmdsynopsis>
     </refsynopsisdiv>
 
+    <refsect1 id='lifecycle'>
+        <title>LIFE-CYCLE</title>
```

Comment:
Let's handle deprecation/removal notice in a different PR (in a slightly different manner).
I wouldn't like to add this message in RHEL9.5(6), as, most probably, `sss_ssh_knownhosts` won't be removed in RHEL9.
Having a downstream only patch to cut this off will be cumbersome.

Reply:
Why not adding the message to RHEL 9.5? It's a good piece of information. As long as `sss_ssh_knownhostsproxy` remains, there will be no compatibility problem during the upgrade to the following RHEL 9.x (x>=5).

Reply:
> Why not adding the message to RHEL 9.5?

I think "soon be removed" would be misleading there, because it won't be removed in RHEL9 at all, and this is years of support.

I propose to have another PR on top of this, that will introduce a ./configure time option to conditionally build **old** "sss_ssh_knownhostsproxy" (not new helper); disabled by default.

Then I'd propose something like:
 - f-41: builds both new and old helpers (+ RN about deprecation and future removal of old helper)
 - f-42: only new helper (+ RN about removal)
 - RHEL10: only new helper starting already beta
 - RHEL9: both helpers in all minor releases

Having wrote this... it's ok to add a warning in this PR, but let's remove "and will soon be removed" and add a few words on the topic why new helper is better (more native integration, better performance wise etc)
And please add the same to `:feature:` RN in the commit message.

Reply:
> > Why not adding the message to RHEL 9.5?
> 
> I think "soon be removed" would be misleading there, because it won't be removed in RHEL9 at all, and this is years of support.

I see your point. I will rephrase it as "in the future", with no time indication.

> I propose to have another PR on top of this, that will introduce a ./configure time option to conditionally build **old** "sss_ssh_knownhostsproxy" (not new helper); disabled by default.

Good idea. 

> Having wrote this... it's ok to add a warning in this PR, but let's remove "and will soon be removed" and add a few words on the topic why new helper is better (more native integration, better performance wise etc) And please add the same to `:feature:` RN in the commit message.

Sure